server.py

```
@@ -0,0 +1,18 @@
+
+provider = FileSystemProvider(Path(__file__).parent / "components")
+
+mcp = FastMCP(name="orion-mcp", providers=[provider])
+
+
+if __name__ == "__main__":
+    if os.getenv("ES_SERVER") is None:
+        print("ES_SERVER environment variable is not set")
+        sys.exit(1)
+    TRANSPORT = "streamable-http"
+    mcp.run(transport=TRANSPORT, host="0.0.0.0", port=3030)
```

Ruff (0.16.3)

[error] Possible binding to all interfaces

(S104)